components/input/Input.js

```diff
@@ -59,8 +69,8 @@ export default class Input extends Component {
   }
 
   componentWillUpdate(props, state) {
-    if (props.onChange && state.value) {
-      props.onChange(state.value);
+    if (props.input && props.input.onChange && state.value) {
+      props.input.onChange(state.value);
```

Comment:
Thanks! This means we will need some kind of helper to create a `redux-form`-esque input field for situations where we don't use it so this API works everywhere. Getting there... 👏